gpii/node_modules/flowManager/src/CloudBasedFlowManager.js

```diff
@@ -211,24 +211,22 @@ gpii.request.flowManager.onOauth2Settings = function (request, handler, dataStor
     }
 };
 
-fluid.defaults("kettle.requests.request.handler.untrustedSettings", {
-    gradeNames: ["fluid.littleComponent", "gpii.request.flowManager.privacyFilteredMatchMakingRequest", "autoInit"],
+fluid.defaults("gpii.flowManager.cloudBased.untrustedSettings.handler", {
+    gradeNames: ["kettle.request.http", "gpii.flowManager.privacyFilteredMatchMakingRequest"],
     invokers: {
-        handle: {
-            funcName: "gpii.request.flowManager.onUntrustedSettings",
+        handleRequest: {
+            funcName: "gpii.flowManager.onUntrustedSettings",
```

Comment:
Should this function be named as ```gpii.flowManager.cloudBased.untrustedSettings.handleRequest``` to be consistent with the naming convention of other request handlers?